packages/features/bookings/lib/handleNewBooking.ts

```diff
@@ -1298,7 +1305,10 @@ async function handler(
         throw new Error(ErrorCode.HostsUnavailableForBooking);
       }
       // Pushing fixed user before the luckyUser guarantees the (first) fixed user as the organizer.
-      users = [...availableUsers.filter((user) => user.isFixed), ...luckyUsers];
+      users = [
+        ...availableUsers.filter((user) => user.isFixed && user.email !== reqBody.teamMemberEmail),
+        ...luckyUsers,
+      ];
```

Comment:
If the team member is a fixed host then remove them from this array so they aren't double booked.